src/haconiwa/tool/parallel_dev.py

```
@@ -0,0 +1,462 @@
+"""Claude Code SDK parallel development functionality."""
+
+import asyncio
+import json
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)